assets/dev/scss/global/_dialog-confirm.scss

```diff
@@ -41,7 +41,13 @@ $e-dialog-confirm-space: 20px;
 		gap: 15px;
 
 		.dialog-button {
-			@extend .elementor-button;
+			@include e-button;
```

Comment:
Have you checked how it is affecting other dialogs?

Reply:
Yapp, hopefully I didn't miss any highly customized ones, but seems ok.